python/foglamp/plugins/north/ocs/ocs.py

```diff
@@ -177,7 +178,11 @@
         "type": "string",
         "default": "int32"
     },
-
+    "destination_type": {
```

Comment:
This is not required. The plugin is an OCS plugin, it does not need to be told it is OCS, and it can never be OMF or Elasticsearch